doc/host-setup.md

```diff
@@ -146,3 +146,39 @@ Checking the current value again will reveal that the helper is now disabled
 # cat /sys/module/drbd/parameters/usermode_helper
 disabled
 ```
+
+## Disable Lvmetad in CentOS 7 and Ubuntu 18.04
+
+Reference: https://access.redhat.com/solutions/2053483
+
+CentOS 7 and Ubuntu 18.04 by default use **lvmetad**,  a metadata caching daemon for LVM. The daemon receives notifications from udev rules. Through these notifications, lvmetad has an up-to-date and consistent image of the volume groups available in the system. lvmetad improves performance of LVM commands by saving them from scanning the disks in every execution. 
+
+`piraeus-ns-node` pod runs lvm commands inside a container without using any lvmetad or udev. And it does not communicate to lvmetad daemon in the operation system. Therefore, users must run `pvscan --cache` in the OS each time to view LVM changes after a Piraeus volume operation. 
```

Comment:
Haven't we considering passing through `/etc/lvm.conf` and `/run/lvm` from host into container to avoid such problems?

Reply:
Yes, I have another incoming PR to support lvmetad just for centos 7 and ubuntu bionic as we also hear from some conservative users who do not want to turn lvmetad off. But so far, disabling lvmetad is the simplest solution and is consistent with the lvm trend in centos 8 and ubuntu focal.

Reply:
Passing `/etc/lvm.conf` and `/run/lvm` is not adequate to enable lvm binary communicate to os-level lvmetad daemon from within `piraeus-ns-node` container.  Only lvm binary lvm2-2.02 binary in centos/ubi 7, ubuntu bionic, and debian strech base images still have support for lvmetad ( they still having `use_lvmetad = 1` in lvm.conf).  `piraeus-sever` is currently based on debian bullseye (presviously buster) image which uses lvm2-2.03 that has deprecated lvmetad support. Therefore, to support lvmetad, `piraeus-server` needs to have a version based on debian stretch (or ubuntu bionic or centos 7) to be able to install lvm2-2.02.  Apart from that, /run/udev is also needed for lvmetad.

Reply:
Unfornately, linbit does not maintain a repo for debian stretch any more. Therefore, I managed to rebase `piraeus-server`to ubuntu bionic image, and successfully had lvm cmd updating lvmetad in os.